mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/impl/SimpleMailboxMessageTest.java

```diff
@@ -59,6 +59,7 @@ public void testSize() {
 
     @Test
     public void testInputStreamSize() throws IOException {
+        @SuppressWarnings("resource")
```

Comment:
You may prefere this fix:
https://github.com/linagora/james-project/pull/193/commits/233a71dc2323e1ca3c91601aa50fe644e2af04b9